### ISSUE TITLE
Does not compile from hex

### ISSUE THREAD

**Issue by lawik** [sasa1977/ci#2]
Elixir: 1.11.3-otp-23
Erlang: 23.0.2

```
lawik@MacBook-Pro webring % mix deps.compile ci
==> ci
Compiling 9 files (.ex)
Generated ci app
spawn: Could not cd to ports/os_cmd

could not compile dependency :ci, "mix compile" failed. You can recompile this dependency with "mix deps.compile ci", update it with "mix deps.update ci" or clean it with "mix deps.clean ci"
```

Maybe this is something to do with Erlang 23.1 but I'm guessing not. Writing this issue while installing 23.1 so intend to try just building the library itself after that but want the right deps and Erlang installs take a bit of time. Figured I'd let you know before I forget.

**Comment by lawik** [comment]
Also happens with Erlang 23.1. The lib compiles fine in isolation.

**Comment by lawik** [comment]
Compiles fine as a path dependency.

**Comment by lawik** [comment]
Found the issue, the PR has the solution.

Hex only packages a certain set of files and directories: https://hexdocs.pm/hex/Mix.Tasks.Hex.Build.html#module-package-configuration

`ports` is not among them

**Comment by lawik** [comment]
Fixed in 0.1.1
